src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -103,11 +103,18 @@ internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<Com
                     }
                 };
 
-                result = await _requestInvoker.RequestServerAsync<CompletionParams, SumType<CompletionItem[], CompletionList>?>(
+                // We want to deserialize to VSCompletionItem type to make sure we don't lose extra information like icons.
+                var response = await _requestInvoker.RequestServerAsync<CompletionParams, SumType<VSCompletionItem[], CompletionList>?>(
```

Comment:
How were we doing this without declaring `var response` previously?

Reply:
We now return `SumType<VSCompletionItem[], CompletionList>?` but that is not directly assignable to `result` which is of type `SumType<CompletionItem[], CompletionList>?>`. Before this it was directly assignable, so we didn't need an intermediate `response` variable.